src/bors/handlers/review.rs

```diff
@@ -176,27 +198,37 @@ PR will need to be re-approved."#,
         .await
 }
 
+async fn notify_of_pushed_pr(
+    repo: &RepositoryState,
+    pr_number: PullRequestNumber,
+    head_sha: CommitSha,
+) -> anyhow::Result<()> {
+    repo.client
+        .post_comment(
+            pr_number,
+            Comment::new(format!(
+                r#":warning: Branch's head was changed to `{}`, the 
```

Comment:
This sounds a bit cryptic, I'd mention the fact that a new commit was pushed to the branch, that will be more obvious.